tests/roles/caller/tasks/main.yml

```diff
@@ -1,10 +0,0 @@
----
-# tasks file for caller
```

Comment:
Frankly, I don't know. @richm any opinions?

Reply:
yes, it is used in tests_include_vars_from_parent.yml to ensure that the role correctly uses vars files, to prevent regressions in vars and vars files handling behavior.